tensorboard/webapp/app_routing/store/app_routing_reducers.ts

```diff
@@ -27,8 +28,15 @@ const reducer = createReducer(
   on(actions.navigating, (state, {after}) => {
     return {...state, nextRoute: after};
   }),
-  on(actions.navigated, (state, {after}) => {
-    return {...state, activeRoute: after, nextRoute: null};
+  on(actions.navigated, (state, {after, afterNamespaceId}) => {
+    return {
+      ...state,
+      activeRoute: after,
+      nextRoute: null,
+      // TODO(bdubois): Remove the null fallback when afterNamespaceId is no
+      // longer optional.
+      activeNamespaceId: afterNamespaceId || null,
```

Comment:
There is new app_routing state: activeNamespaceId that we can use to populate beforeNamespaceId on navigated() events.